app/ui-react/syndesis/src/modules/integrations/components/editor/endpoint/WithConfigurationForm.tsx

```diff
@@ -156,6 +160,11 @@ export const WithConfigurationForm: React.FunctionComponent<IWithConfigurationFo
       </ConfigurationForm>
     );
   }
+
+  const webhookDefinitionOverride = isWebHook
+    ? applyErrorKeysToForm(action, props.errorKeys!)
```

Comment:
So this won't avoid ENTESB-13527 I think, as the `action` here won't have the generated `defaultValue` property.  I'll suggest what you may need to do below.